dynamicprefixvici.c

```diff
@@ -35,29 +45,80 @@ static void ParseCommandLineArguments(int argc, char** argv, CommandLineArgument
  */
 static void Usage();
 
+/**
+ * @brief Converts a command line argument to an integer
+ * Prints an error message if the argument is invalid and exits
+ * 
+ * @param value 
+ * @param optionname 
+ * @return int 
+ */
+static int parse_option_to_integer(char* value, char* optionname);
+
+/**
+ * @brief Checks whether the string only contains the character specified
+ * Returns true if the string only contains the characters
+ * @param string 
+ * @param characters 
+ * @return true 
+ * @return false 
+ */
+static bool check_only_contains(char* string, char* characters);
+
+/**
+ * @brief Check whether something is a valid IPv6 address
+ * 
+ * @param adres 
+ * @return true 
+ * @return false 
+ */
+static bool check_adres(char* adres);
+
+/**
+ * @brief Checks wheters de input arguments are all valid
+ * 
+ * @param argumenten 
+ * @return true 
+ * @return false 
+ */
+static bool check_validity_of_arguments(CommandLineArguments* argumenten);
```

Comment:
Deze geen void maken en exiten in de functie zelf ipv de return value te checken?